cloud-common/src/main/java/com/sequenceiq/cloudbreak/cloud/store/InMemoryStateStore.java

```diff
@@ -1,27 +1,40 @@
 package com.sequenceiq.cloudbreak.cloud.store;
 
+import com.sequenceiq.cloudbreak.cloud.scheduler.PollGroup;
+
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-import com.sequenceiq.cloudbreak.cloud.scheduler.PollGroup;
-
 public final class InMemoryStateStore {
 
-    private static final Map<Long, PollGroup> STATE_STORE = new ConcurrentHashMap<>();
+    private static final Map<Long, PollGroup> STACK_STATE_STORE = new ConcurrentHashMap<>();
+    private static final Map<Long, PollGroup> CLUSTER_STATE_STORE = new ConcurrentHashMap<>();
```

Comment:
I don't think that, this is a good direction. Since if we follow this concept then we need to put the also the Swarm polling state separately, which means that the clients needs to always execute the following sequence:
if (cluster != POLLABLE && orchestrator != pollable  && cluster != pollable) ...

it would be better to handle this uniformly and the client needs to call just isCancelled method. Of course this means that some common key needs to be introduced.